src/vm/appdomain.cpp

```diff
@@ -4313,8 +4313,8 @@ static const char *fileLoadLevelName[] =
     "LOADLIBRARY",                        // FILE_LOAD_LOADLIBRARY
     "POST_LOADLIBRARY",                   // FILE_LOAD_POST_LOADLIBRARY                
     "EAGER_FIXUPS",                       // FILE_LOAD_EAGER_FIXUPS
-    "VTABLE FIXUPS",                      // FILE_LOAD_VTABLE_FIXUPS
     "DELIVER_EVENTS",                     // FILE_LOAD_DELIVER_EVENTS
```

Comment:
Why this order worked fine for .NET Framework, and it does not work for .NET Core?

Reply:
On .NET Framework, we would correctly load into the same app-domain. For .NET Core, it seems that the an assembly needs to be loaded at least to the FILE_LOAD_DELIVER_EVENTS level for a dependent AssemblySpec to consider it loaded enough to use its ALC to load the assembly.